gemini/mcp/mcp_orchestration_app/src/gemini_server.py

```diff
@@ -0,0 +1,317 @@
+"""
+Copyright 2025 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import logging
+import os
+import sys
+
+import nest_asyncio
+from mcp.server.fastmcp import FastMCP
+from google import genai
+from google.api_core import exceptions as google_exceptions
+from google.cloud import translate_v3 as translate
+from google.genai import types
+
+# Apply nest_asyncio
+nest_asyncio.apply()
+
+# Configure logging - Set default level to INFO
+logging.basicConfig(
+    level=logging.ERROR,
+    format="%(asctime)s - %(levelname)s [%(name)s] - %(message)s",
+)
+
+# --- Suppress Verbose Google API Logs ---
+# Set levels for specific noisy loggers
+logging.getLogger("google.api_core").setLevel(logging.WARNING)
+logging.getLogger("google.auth").setLevel(logging.WARNING)
+logging.getLogger("google.generativeai").setLevel(logging.WARNING)
+logging.getLogger("google.cloud").setLevel(logging.WARNING)
+logging.getLogger("urllib3").setLevel(logging.WARNING)
+logging.getLogger("httpcore").setLevel(logging.WARNING)
+logging.getLogger("httpx").setLevel(logging.WARNING)
+
+# --- Configuration ---
+GOOGLE_PROJECT_ID = os.environ.get(
+    "GOOGLE_CLOUD_PROJECT", "cloud-project-test"
```

Comment:
```suggestion
    "GOOGLE_CLOUD_PROJECT"
```

Remove this extra default value, this should fail if there is not a project set.